client/src/App.tsx

```diff
@@ -5,7 +5,7 @@ import styled, { ThemeProvider } from 'styled-components';
 import Navbar from './components/navigation/Navbar';
 import Footer from './components/navigation/Footer';
 import HomePage from './components/pages/HomePage/HomePage';
-import RelatedPage from './components/pages/RelatedPage/RelatedPage';
+// import RelatedPage from './components/pages/RelatedPage/RelatedPage';
```

Comment:
I think you can leave this in? Shouldn't be any harm as long as we dont show the links anywhere